addons/api/package.json

```diff
@@ -80,5 +79,10 @@
   },
   "ember-addon": {
     "configPath": "tests/dummy/config"
+  },
+  "lint-staged": {
```

Comment:
I didn't include these commands in our scripts because they're one off and not that useful out of the context of `lint-staged` (they get called with the files from lint-staged but they don't specify any by themselves)

Do you have any thoughts @randallmorey? Would you still prefer these commands be in scripts?